package.json

```diff
@@ -13,6 +13,9 @@
   },
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.9.0",
+    "@emotion/styled": "^11.8.1",
+    "@mui/material": "^5.6.2",
```

Comment:
non-blocking: Do we really need this package (another one set of components) similar to blueprintjs and internal ui-toolkit? Wouldn't it be better to keep one "component facilitator" than many of them in one repo?

Reply:
moreover, tailwind.css alone offers many possibilities